internal/tool/single/single.go

```diff
@@ -36,14 +36,32 @@ var (
 		},
 	}
 
+	purgeSpec = &tool.PurgeSpec{
+		Tool:  "weaver single",
+		Paths: []string{filepath.Join(must(files.DefaultDataDir()), "single_registry")},
+	}
+
 	Commands = map[string]*tool.Command{
 		"status":    status.StatusCommand("weaver single", defaultRegistry),
 		"dashboard": status.DashboardCommand(dashboardSpec),
 		"metrics":   status.MetricsCommand("weaver single", defaultRegistry),
 		"profile":   status.ProfileCommand("weaver single", defaultRegistry),
+		"purge":     tool.PurgeCmd(purgeSpec),
 	}
 )
 
+// must is a generic implementation of the Go must idiom [1, 2]. It panics if
```

Comment:
can we put this method somewhere in a shared directory? Right now is duplicated among single, multi.

Reply:
Done.